Report API for Developer.md

```diff
@@ -1,7 +1,9 @@
 ## 1. Integration Instructions
 
 ### 1.1 Request URL
-https://pa-report-en.zplayads.com
+Please choose the right interface address according to the settlement currency of your account:
+When settlement currency is Chinese Yuan:https://pa-report.zplayads.com
```

Comment:
If the settlement currency is Chinese Yuan, please use this interface addres: https://pa-report.zplayads.com